docs/grid_model/network_subnetwork.md

```diff
@@ -95,6 +95,26 @@ When defining the model, the user has to specify how the different equipment con
 - [Identifiable Short-Circuit](extensions.md#identifiable-short-circuit)
 - [Slack Terminal](extensions.md#slack-terminal)
 
+## Area
+[![Javadoc](https://img.shields.io/badge/-javadoc-blue.svg)](https://javadoc.io/doc/com.powsybl/powsybl-core/latest/com/powsybl/iidm/network/Area.html)
+
+An Area represents geographical zone defined by the [voltage levels](#voltage-level) that it contains and by its boundaries, that can be terminals of equipments or `Boundary` objects from [dangling lines](#dangling-line).
+
+Areas can represent various granularities depending on their types. For instance: control areas, bidding zones, countries...
+
+A [voltage level](#voltage-level) can belong to several Areas, as long as they all have different types.
+
+The powerflow through the area boundaries can be summed and used to perform calculations for area interchange control.
```

Comment:
Please add convention. What is the definition of ac net interchange and dc net interchange with and without any boundaries in the list.

Reply:
The doc has been updated in d40f8ba4d8776fff3162f66b8a178e1290cb0456